src/components/Layout/NetworkSelect/NetworkSelect.tsx

```diff
@@ -34,73 +34,26 @@ const WrappedDropdown = (props: DropdownProps) => {
   const { children, className, ...rest } = props;
   return (
     <div className={className}>
-      <Dropdown className="rounded-md py-1 pl-4 p-2 border border-gray-300 bg-white" {...rest}>
-        {children}
-      </Dropdown>
+      <Dropdown {...rest}>{children}</Dropdown>
     </div>
   );
 };
 
-const StyledDropdown = styled(WrappedDropdown)`
-  display: inline-block;
-  min-width: 12.5rem;
-  font-size: 0.87rem;
-
-  span.select-msg {
-    padding: 0.25rem 0.7rem;
-    color: darkgrey;
-    cursor: default;
-  }
-`;
-
 /**
  * Label for the items of the dropdown list
  */
 const DropdownItemLabel: FunctionComponent<DropdownItemLabelProps> = ({ className, active, network }) => {
   return (
-    <div className={className} data-testid={`network-select-dropdown-label-${network.chainId}`}>
-      <img className="network-icon" src={network.iconImage} alt={network.label} />
-      <span className="label">{network.label}</span>
-      {active ? <span className="active" /> : null}
+    <div className={className}>
+      <div className="flex items-center" data-testid={`network-select-dropdown-label-${network.chainId}`}>
+        <img className="mr-2 w-5 h-5 rounded-full" src={network.iconImage} alt={network.label} />
+        <span className="py-2 hover:text-cerulean transition-colors duration-200 ease-out w-full">{network.label}</span>
+        {active ? <span className="m-1 active-icon justify-self-end" /> : null}
+      </div>
```

Comment:
lets inline the active class names here, so no need to declare at tailwind.css level